.dockerignore

```diff
@@ -4,4 +4,7 @@ deps/
 .elixir_ls/
 .git/
 .github/
-helm/
\ No newline at end of file
+helm/
+apps/service_define/assets/node_modules/
+apps/service_define/assets/.cache/
+apps/service_define/assets/build/
```

Comment:
i take it there _are_ assets we want to upload so you can't just omit the entire `apps/service_define/assets/` dir?

Reply:
This is correct. assets/ contains all the UI code, and these a just a few folders that get generated that we don't want.